les/redemption-winnercheque-v1.html

```diff
@@ -46,29 +46,32 @@ <h5 id="winnerChequeMonetaryAmountInWords"><a href="#winnerChequeMonetaryAmountI
     <h5 id="winnerChequeTransactionId"><a href="#winnerChequeTransactionId">transactionId</a></h5>
     The transaction id of transfer stable coins from casino to player during creating winner cheque.
   </li>
-</ul>
-
-<h4 id="player"><a href="#player">player</a></h4>
-The player that has winner cheque.
-<ul>
-  <li>
-    <h5 id="playerId"><a href="#playerId">playerId</a></h5>
-    Unique identifier belonging to the player.
-  </li>
-  <li>
-    <h5 id="cryptoAddress"><a href="#cryptoAddress">cryptoAddress</a></h5>
-    The crypto address of the player.
-  </li>
   <li>
-    <h5 id="name"><a href="#name">name</a></h5>
-    The name of the player.
-  </li>
-  <li>
-    <h5 id="middleName"><a href="#middleName">middleName</a></h5>
-    The middle name of the player.
-  </li>
-  <li>
-    <h5 id="surname"><a href="#surname">surname</a></h5>
-    The surname of the player.
+    <h5 id="player"><a href="#player">player</a></h5>
+    The player that has winner cheque.
+    <ul>
+      <li>
+        <h6 id="playerId"><a href="#playerId">playerId</a></h6>
```

Comment:
Not sure of that is your intent @mehmetbicak. But for now include a player VC as part of the Winnercheque, so that an external party does not need to seperately get the VC associated with the cheque

Reply:
I am doing this.
But it would be great to have your feedback to agree and not miss some points. If you have time tomorrow or Friday, I can demo you the latest changes.